assets/queries/ansible/gcp/compute_instance_is_publicly_accessible/query.rego

```diff
@@ -0,0 +1,25 @@
+package Cx
+
+CxPolicy [result ]  {
+	document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+
+  task["google.cloud.gcp_compute_instance"].network_interfaces[_].access_configs
+
+  result := {
+            "documentId": document.id,
+        	  "searchKey": sprintf("name=%s.{{google.cloud.gcp_compute_instance}}.network_interfaces", [task.name]),
```

Comment:
should point directly to access_configs, change to:
sprintf("name=%s.{{google.cloud.gcp_compute_instance}}.network_interfaces.access_configs", [task.name]),
and fix positive expected result